### PR TITLE
v0.14.0 Release

### PR DESCRIPTION
**Pull request by Monte9**
### Release notes:
https://github.com/react-native-training/react-native-elements/releases/tag/untagged-c5258711f1d5839f9efa

### Expo App:
https://expo.io/@monte9/react-native-elements-app

### CODE REVIEW

File: src/card/Card.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import PropTypes from 'prop-types';
```
Comment (binoy14) [inline src/card/Card.js]: There is a duplicate file in the containers, if this is being moved than let's delete the container folder

**Comment by Monte9** [comment]
Removed peerDependency of `react-native-vector-icons` on package.json. See [this commit](https://github.com/react-native-training/react-native-elements/pull/438/commits/681e9a344fe5f4889fb4f4931b6561ed9faf28c4)

The `example` app, for instance, uses expo and we don't explicitly need to install rn-vector-icons for an expo project.. and in such cases, it shows the following warning:

```npm WARN react-native-elements@0.14.0 requires a peer of react-native-vector-icons@^4.2.0 but none was installed.```

@binoy14 thoughts on this?

File: src/card/Card.js (hunk 1 above)
Comment (Monte9) [inline src/card/Card.js]: Done. ✅

**Comment by binoy14** [comment]
@Monte9 It might be fine but it won't let the user know when they are not using CRNA or expo

**Pull request merged**